tests/test_compose.py

```diff
@@ -19,13 +19,16 @@ def test_compose():
         # transform must be callable or a dict
         Compose('LoadImageFromFile')
 
-    target_keys = ['img', 'img_metas']
+    target_keys = ['img', 'img_rename', 'img_metas']
```

Comment:
Where does `img_rename` come from?

Reply:
This is to test a new function in "Collect" that it can rename the collected items. A possible use of this in the 2d-to-3d task is to choose model input from 2d pose annotation projected 3d pose.